packages/clerk-js/src/v4/SignIn/SignInFactorTwo.tsx

```diff
@@ -21,63 +44,32 @@ export function _SignInFactorTwo(): JSX.Element {
     }
   }, []);
 
-  return <SignInFactorTwoPhoneCodeCard />;
+  if (!currentFactor) {
```

Comment:
🙃 This can be removed since there is a default clause in the following switch statement